docs/releasenotes/7.2.0.rst

```diff
@@ -37,6 +37,14 @@ are now read as just a single bytestring.
 Deprecations
 ^^^^^^^^^^^^
 
+Image.show command parameter
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. deprecated:: 7.2.0
```

Comment:
Not sure if we need this line in 7.2.0 release notes

https://pillow--4723.org.readthedocs.build/en/4723/releasenotes/7.2.0.html#deprecations

Reply:
Yep, removed